yb-voyager/cmd/analyzeSchema.go

```diff
@@ -584,32 +575,26 @@ func createSqlStrInfoArray(path string, objType string) []sqlInfo {
 		singleLine += curr + " "
 		singleString += curr + "\n"
 
-		if codeBlock {
-			// Assuming that both the dollar quote strings will not be in same line
-			if dollarQuoteFlag == 0 {
-				if idx := dollarQuoteRegex.FindStringSubmatch(curr); idx != nil {
-					dollarQuoteFlag = 1 //denotes start of the code/body part
-					outerDollarQuotedStr = idx[0]
-				} else if strings.Contains(curr, ";") { // in case, there is no body part
-					//one liner sql string created, now will check for obj count and report cases
-					processCollectedSql(path, &singleLine, &singleString, objType, &sqlInfoArr, &reportNextSql)
-				}
-			} else if dollarQuoteFlag == 1 {
-				if strings.Contains(curr, outerDollarQuotedStr) {
-					dollarQuoteFlag = 2 //denotes end of code/body part
-				}
+		// Assuming that both the dollar quote strings will not be in same line
+		if dollarQuoteFlag == 0 {
+			if idx := dollarQuoteRegex.FindStringSubmatch(curr); idx != nil {
+				dollarQuoteFlag = 1 //denotes start of the code/body part
+				outerDollarQuotedStr = idx[0]
+			} else if strings.Contains(curr, ";") { // in case, there is no body part
+				//one liner sql string created, now will check for obj count and report cases
+				processCollectedSql(path, &singleLine, &singleString, objType, &sqlInfoArr, &reportNextSql)
```

Comment:
Let's rename `singleLine` as `stmt` and `singleString` as `formattedStmt` to align it with the `sqlInfo` definition.